docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,296 @@
+#####################################
+Setting transpiler optimization level
+#####################################
+
+This guide shows you how to use the :attr:`~qiskit.transpile.optimization_level` 
+parameter with :meth:`~qiskit.compiler.transpile`.
+:attr:`~qiskit.transpile.optimization_level` helps you to optimize your quantum circuit.
+This parameter takes an integer which can be a value between 0 and 3,
+where the higher the number the more optimized the result.
+You can find more information about this parameter
+`here <https://qiskit.org/documentation/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.html#Preset-Pass-Managers>`__.
+
+Initialize the quantum circuit
+==============================
+
+The effect of setting the :attr:`~qiskit.transpile.optimization_level` will differ depending on the backend you are using.
+For this example you will exploring the `CSWAP <https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.cswap.html>`__ gate,
+a three qubit gate which will be transpiled into one and two qubit gates.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister 
+    from qiskit.compiler import transpile
+    from qiskit.providers.fake_provider import FakeQuito #fake backend ibmq_quito
+
+    backend = FakeQuito() # For this example the quito fake backend is used
+
+    qr = QuantumRegister(3) #Initialize the Quantum Register 
+    qc = QuantumCircuit(qr) # Initialize the quantum circuit 
+    
+    qc.cswap(qr[0],qr[1],qr[2]) #add the cswap gate to the quantum circuit
```

Comment:
If you don't use the registers in a workflow later, you could also just drop them and initialize the circuit directly with the number of qubits. 
```suggestion
    qc = QuantumCircuit(3) 
    
    qc.cswap(0, 1, 2) #add the cswap gate to the quantum circuit
```

Reply:
I agree! I'll do the changes